src/app/irena/styles/pages/igre.css

```
@@ -0,0 +1,658 @@
+.modal-bgg-link a {
+  color: var(--accent);
+  font-size: 13px;
+  text-decoration: underline;
+}
+
+.modal-video {
+  position: relative;
+  width: 100%;
+  aspect-ratio: 16 / 9;
+
+  margin-top: 14px;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)